### PR TITLE
feat: activate app with timeout

### PR DESCRIPTION
**Pull request by agentran**
## Change list

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New feature (non-breaking change which adds value to the project)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Documentation
- [ ] Have you proposed a file change/ PR with appium to update documentation? 
## Integration tests
- [ ] Have you provided integration tests to pass against the beta version of appium? (for Bugfix or New feature)

## Details

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

**Comment by agentran** [comment]
@Dor-bl Please note that the CLA was already signed before, I don't know why it does not update.

**Comment by Dor-bl** [comment]
From where did you created the PR?
Maybe there is a conflict with the username you use when pushing the changes. 
@akinsolb maybe you came across this in the past?

**Comment by agentran** [comment]
@Dor-bl I used the same user, created a fork merge request using github. (same as I did before)

**Comment by Dor-bl** [comment]
Check this out please: https://stackoverflow.com/questions/65230043/unable-to-commit-to-github-getting-commit-didnt-pass-easycla-test

**Comment by agentran** [comment]
@Dor-bl Resolved.

File: test/integration/Android/Device/AppTests.cs [hunk 1]
```diff
@@ -43,7 +43,16 @@ public void CanActivateAppTest()
             //Verify the expected app was activated
             Assert.DoesNotThrow(() => _driver.FindElement(MobileBy.Id(IntentAppElement)));
         }
+        
+        [Test]
+        public void CanActivateAppWithTimeoutTest()
+        {
+            //Activate an app to foreground
+            Assert.DoesNotThrow(() => _driver.ActivateApp(IntentAppPackageName, TimeSpan.FromSeconds(20)));
```
Comment (Dor-bl) [inline test/integration/Android/Device/AppTests.cs:51]: https://github.com/appium/dotnet-client/pull/576 This Is the removal of IntentAppPackageName from the AppTests.
So you will need to take it into consideration

**Review by Dor-bl — CHANGES_REQUESTED** [review]

File: test/integration/Android/Device/AppTests.cs (hunk 1 above)
Comment (agentran) [reply test/integration/Android/Device/AppTests.cs]: resolved

**Comment by agentran** [comment]
hey @Dor-bl, can we please approve this pr?

**Comment by Dor-bl** [comment]
Sure, just you have one open review. Once you replay I will approve

**Comment by agentran** [comment]
which review is open?

**Comment by Dor-bl** [comment]
This one https://github.com/appium/dotnet-client/pull/579#discussion_r1065777689

**Comment by agentran** [comment]
hey @Dor-bl
looks like there is an issue with the test suite itself, at least locally tests are not working.
also I can't seem to find that comment on the PR itself, github is being weird about it 🤔

**Comment by Dor-bl** [comment]
@agentran can you open a new issue for the test suite that is not working for iOS? 
I'll merge this PR.

File: test/integration/IOS/Device/AppTests.cs [hunk 2]
```diff
@@ -46,6 +46,16 @@ public void CanActivateAppTest()
             Assert.DoesNotThrow(() => _driver.FindElement(MobileBy.AccessibilityId(IosTestAppElement)));
         }
 
+        [Test]
+        public void CanActivateAppWithTimeoutTest()
+        {
+            //Activate an app to foreground
+            Assert.DoesNotThrow(() => _driver.ActivateApp(IosTestAppBundleId, TimeSpan.FromSeconds(20)));
```
[the remark this answers is not in the document]
Comment (Dor-bl) [reply test/integration/IOS/Device/AppTests.cs:53]: @agentran will be handled on a separate PR to fix iOS AppTests.

**Pull request merged**